test/appium/tests/critical/test_wallet.py

```diff
@@ -0,0 +1,190 @@
+        self.sender, self.receiver = transaction_recipients['J'], transaction_recipients['K']
+        self.sender_username, self.receiver_username = 'sender', 'receiver'
+        self.loop.run_until_complete(
+            run_in_parallel(((self.sign_in_1.recover_access, {'passphrase': self.sender['passphrase'],
+                                                              'username': self.sender_username}),
+                             (self.sign_in_2.recover_access, {'passphrase': self.receiver['passphrase'],
+                                                              'username': self.receiver_username}))))
+        self.home_1, self.home_2 = self.sign_in_1.get_home_view(), self.sign_in_2.get_home_view()
+        self.wallet_1 = self.home_1.wallet_tab.click()
+        self.wallet_2 = self.home_2.wallet_tab.click()
+
+    # ToDo: Add verification of Activity tabs when the feature is ready in the next 2 tests:
+
+    def _get_balances_before_tx(self):
+        sender_balance = self.network_api.get_balance(self.sender['address'])
+        receiver_balance = self.network_api.get_balance(self.receiver['address'])
+        self.wallet_1.just_fyi("Getting ETH amount in the wallet of sender before transaction")
+        eth_amount_sender = self.wallet_1.get_asset(asset_name='Ether').get_amount()
+        self.wallet_2.just_fyi("Getting ETH amount in the wallet of receiver before transaction")
+        self.wallet_2.get_account_element().click()
+        eth_amount_receiver = self.wallet_2.get_asset(asset_name='Ether').get_amount()
+        return sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver
+
+    def _check_balances_after_tx(self, amount_to_send, sender_balance, receiver_balance, eth_amount_sender,
+                                 eth_amount_receiver):
+        try:
+            self.network_api.wait_for_balance_to_be(address=self.sender['address'],
+                                                    expected_balance=sender_balance - amount_to_send)
+        except TimeoutException:
+            self.errors.append("Sender balance was not updated")
+        try:
+            self.network_api.wait_for_balance_to_be(address=self.receiver['address'],
+                                                    expected_balance=receiver_balance + amount_to_send)
+        except TimeoutException:
+            self.errors.append("Receiver balance was not updated")
+
+        self.wallet_1.just_fyi("Getting ETH amount in the wallet of sender after transaction")
+        new_eth_amount_sender = self.wallet_1.get_asset(asset_name='Ether').get_amount()
+        self.wallet_2.just_fyi("Getting ETH amount in the wallet of receiver after transaction")
+        self.home_2.wallet_tab.click()
+        new_eth_amount_receiver = self.wallet_2.get_asset(asset_name='Ether').get_amount()
+        exp_amount_sender = eth_amount_sender - amount_to_send
+        if new_eth_amount_sender > exp_amount_sender:
```

Comment:
are we cure that in this place cannot be delay in updating the balance inside the app?

Reply:
Not yet, I'll add it later